pgconfeu-2024.mdx

```diff
@@ -0,0 +1,309 @@
+---
+title: 'What do I look forward to at PGConf.EU in Athens?'
+description: '14th PostgreSQL Conference in Europe is taking place in Athens, Greece from 22-25 October 2024. In this blog post, I will be sharing what I look forward to at PGConf.EU 2024 including a great keynote, extension ecosystem summit, Xata dinner and more.'
+image:
+  src: https://us-east-1.storage.xata.sh/uq2d57g5hsgdo2ih1nnau2kdbpi6l05c
+  alt: 'PGConfEU 2024 in Athens'
+author: Gulcin Yildirim Jelinek
+authorEmail: gulcin@xata.io
+date: 10-21-2024
+tags:
+  [
+    'postgres',
+    'pgconfeu',
+    'athens',
+    'xata',
+    'postgresql',
+    'postgres-dinner',
+    'postgres-europe',
+    'europe',
+    'extensions',
+    'extension-development',
+    'extension-summit',
+    'diversity-task-force',
+    'pgroll',
+    'schema-migration',
+    'zero-downtime',
+    'zero-downtime-deployment',
+    'zero-downtime-schema-changes',
+    'zero-downtime-migrations',
+    'instant-rollback',
+    'rollback',
+    'uptime',
+    'pgconf',
+    'pgconf-eu',
+    'pgconf-europe',
+    'pgconf-2024',
+    'pgconf-2024-athens',
+    'prague',
+    'berlin',
+    'lisbon',
+    'tallinn',
+    'fosdem'
+  ]
+published: true
+slug: pgconfeu-2024
+ogImage: https://us-east-1.storage.xata.sh/uq2d57de9bli5qd91j16pjps7pfto8vk
+---
+
```

Comment:
```suggestion
As [PGConf.EU](https://pgconf.eu/) kicks off this week in Athens, I wanted to share why this event is special to me and thanks to Xata I'm able to once again attend and be involved in many initiatives. 

You can find me talking at the [Ecosystem Summit](#ecosystem-summit), chairing the Postgres Europe Diversity Committee, or hosting a Xata dinner (grab your RSVP below). But also if you're planning to attend the conference, don't miss out on this talk: Postgres schema migrations using the expand/contract pattern which is hosted by pgroll creator / Xata Software Engineer, Andrew Farries.

But first, a little bit of history...
```

Reply:
Thanks Cezz. 

I would not use "thanks to Xata I am able to once again.." part at all. 

[Extension Ecosystem Summit] 
Not chairing the diversity committee but part of the committee
Also I would not invite for Andrew's call this early. Xata dinner is enough to end it.

Reply:
The name is [Extension Ecosystem Summit]. 

The rest is great.